src/gatsby/models/authors/createSchemaCustomization.js

```diff
@@ -30,8 +29,17 @@ async function createAuthorSchemaCustomization(api) {
         links: '[AuthorLink]',
         slug: 'String',
         avatar: {
-          type: 'ImageSharp',
-          resolve: resolveAuthorAvatar
+          type: 'File',
+          resolve: (source, args, context) =>
+            context.nodeModel.findOne({
+              type: 'File',
+              query: {
+                filter: {
+                  sourceInstanceName: { eq: 'avatars' },
+                  relativePath: { eq: source.avatar }
+                }
+              }
+            })
```

Comment:
Simpler, closer to built-in query for linking avatars/pictures to files which doesn't need a whole module dedicated to it.